src/Path.hs

```diff
@@ -103,22 +105,26 @@ parseJSONWith f x =
        Left e -> fail (show e)
 {-# INLINE parseJSONWith #-}
 
--- | Exception when parsing a location.
-data PathParseException
+-- | Exceptions during path operations
+data PathException
   = InvalidAbsDir FilePath
   | InvalidRelDir FilePath
   | InvalidAbsFile FilePath
   | InvalidRelFile FilePath
   | Couldn'tStripPrefixDir FilePath FilePath
```

Comment:
`Couldn'tStripPrefixDir` is a bit verbose and generic. Shall we rename this to something like `NotAPrefix`? It indicates more clearly that path1 is not a prefix of path2.

This should be safe to change since the exception constructors were not exposed till now.